src/bust.py

```diff
@@ -295,12 +322,21 @@ async def ffmpeg_post_hook(e: Exception = None):
         play_lock = asyncio.Lock()
         # Acquire the lock during playback so that on release, play_song() returns
         await play_lock.acquire()
-        self.voice_client.play(
-            FFmpegPCMAudio(
-                local_filepath, options=f"-filter:a volume={config.VOLUME_MULTIPLIER}"
-            ),
-            after=ffmpeg_post_hook,
-        )
+        if(timestamp > 0):
+            self.seek_and_convert_to_opus(timestamp, local_filepath)
+
+            self.voice_client.play(
+                FFmpegOpusAudio(
+                    self.temp_audio_file, options=f"-filter:a volume={config.VOLUME_MULTIPLIER}"),
```

Comment:
I think we should be able to keep the actual `self.voice_client.play` call with all its options outside the if statement (avoiding duplication) if we do something like

```python
file_to_play = local_filepath
if (timestamp > 0):
    self.seek_and_convert_to_opus(timestamp, local_filepath)
    file_to_play = self.temp_audio_file
```
and then put the voice_client.play after